src/_gettsim/parameters/sozialv_beitr.yaml

```diff
@@ -274,6 +274,15 @@ beitr_satz:
     reference: >-
       (arbeitsl_v:) Art. 2 Nr. 15 G. v. 18.12.2018 BGBl. I S. 2651
       (ges_krankenv mean_zusatzbeitrag:) BAnz AT 31.10.2022 B5
+  2023-07-01:
```

Comment:
I think we need `deviation_from: previous` here such that parameters for the other social insurances